rllib/models/modelv2.py

```diff
@@ -223,6 +232,26 @@ def from_batch(self, train_batch, is_training=True):
             i += 1
         return self.__call__(input_dict, states, train_batch.get("seq_lens"))
 
+    def get_view_requirements(self, is_training=False):
+        """Returns a list of ViewRequirements for this Model (or None).
+
+        A ViewRequirement object tells the caller of this Model, which
```

Comment:
Shall we try a namedtuple or class instead of raw dicts here? Since this is a python only API, no need to be Json compatible.

Reply:
Used to be a dataclass. I can change it to a `Dict[str, dataclass-object]`, where str is the column name in the `input_dict`.

Reply:
done